### DISCUSSION TITLE
How to add a word to every dictionary for all languages?

### DISCUSSION THREAD

**Discussion by aloispichler** [valentjn/vscode-ltex#308] [Q&A] [answered]
The setting for `ltex.dictionary` is language specific.
However, there are words (as _B-spline_, say, or names as _Java_ or _Einstein_, or other stuff like _%appdata%\Code\User\settings.json_) which are not spelling errors in any language.
Is there a way to add _Einstein_, say, to every dictionary?

**Comment by valentjn** [comment, marked as answer]
No, there is no way. It wouldn't make much sense either, IMO. For example, &ldquo;B-spline&rdquo; is English, but &ldquo;B-Spline&rdquo; is German. It's usually a spelling error in the other language, respectively. Also, names may be spelled differently in different languages (e.g., English &ldquo;Cologne&rdquo; and &ldquo;Gorbachev&rdquo; vs. German &ldquo;Köln&rdquo; and &ldquo;Gorbatschow&rdquo;). Finally, stuff like `%appdata%\Code\User\settings.json` should be written as inline code (backticks in Markdown, just like I just did in this sentence). LT<sub>E</sub>X will then automatically replace it with a dummy that doesn't issue a diagnostic.

**Reply by lorenzleutgeb** [reply]
However, there are languages that are sufficiently similar to warrant sharing a dictionary. For example, there are words that are spelled exactly the same in `de-AT` and `de-DE`, as well as in `en-GB` and `en-US`. I'd like to avoid duplication, and have `de-*` re-use a `de` dictionary, and `en-*` to re-use a `en` dictionary, you get the idea...

For a specific set of languages, one can do this:

```json
"ltex.dictionary": {
    "en-US": [":ltex.dictionary.en.txt", ":ltex.dictionary.en-US.txt", ":ltex.dictionary.txt"],
    "en-GB": [":ltex.dictionary.en.txt", ":ltex.dictionary.en-GB.txt", ":ltex.dictionary.txt"],
}
```
